source/SIL.AppBuilder.Portal.Frontend/src/ui/components/header/user-dropdown.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+import { withRouter, RouteComponentProps  } from 'react-router-dom';
+import {
+  Container, Menu, Button,
+  Dropdown, Image, Icon
+} from 'semantic-ui-react';
+
+
+import { deleteToken } from '@lib/auth0';
+import './header.scss';
+
+export type IProps =
+  & { toggleSidebar: () => void }
+  & RouteComponentProps<{}>;
+
+class UserDropdown extends React.Component<IProps> {
+
+  handleSignOut = () => {
+    const { history } = this.props;
+
+    deleteToken();
+    history.push('/login');
+  }
+
+  render() {
+    const { history, toggleSidebar } = this.props;
```

Comment:
you are not using toggleSidebar

Reply:
yeah, it might be needed depending on how the behavior ends up working out as we implement things in the sidebar. 

we should probably have a PR that throws linting errors on unused variables though.